### PR TITLE
Partner Engagements tab max width

### PR DESCRIPTION
**Pull request by rdonigian**
https://seed-company-squad.monday.com/boards/5989610236/views/132288519/pulses/6786242569

### CODE REVIEW

File: src/scenes/Partners/Detail/PartnerDetail.tsx [hunk 1]
```diff
@@ -205,7 +205,7 @@ const PartnerTabs = (props: PartnerViewEditProps) => {
         <Tab label="Engagements" value="engagements" />
         <Tab label="Notes" value="notes" />
       </TabList>
-      <Paper sx={{ maxWidth: 'lg' }}>
+      <Paper sx={{ maxWidth: filters.tab === 'engagements' ? 'auto' : 'lg' }}>
```
Comment (CarsonF) [inline src/scenes/Partners/Detail/PartnerDetail.tsx:208]: I'd rather not do this. It breaks the open/close principle.
Can we do
```ts
<Paper sx={(theme) => ({
  '--maxWidth': theme.breakpoints.values.lg,
  maxWidth: 'var(--maxWidth)',
}}
```
And in the engagements tab
```ts
sx={{
  ':root': {
    '--maxWidth': 'initial',
  }
}}
```

**Review by CarsonF — CHANGES_REQUESTED** [review]

**Comment by CarsonF** [comment]
Does this work?

**Comment by rdonigian** [comment]
It seemed like it but after a second glance apparently all the tabs are getting set to full width, so just working through on how to properly address it.

**Comment by CarsonF** [comment]
It appears that this wouldn't work like I was thinking it would. I think we need to shift the `<Paper>` down to each tab. I originally moved it up here to keep it DRY, but not we can't customize.
I took the opportunity to fix this and a handful of other things on this page: #1535

**Comment by rdonigian** [comment]
This is been solved by https://github.com/SeedCompany/cord-field/pull/1535

**Pull request closed without merging**